### DISCUSSION TITLE
How to display the badge of the build status in the `readme.md` file?

### DISCUSSION THREAD

**Discussion by kimmy-wang** [vercel/next.js#12903] [Help] [answered]
How to display the badge of the build status in the `readme.md` file? Similar to the `netlify` badge.

**Comment by vxcozy** [comment]
has this ever been answered anywhere?

**Comment by jamesmosier** [comment, marked as answer]
There is no badge currently for this for Vercel, but following [this comment](https://github.com/vercel/vercel/discussions/4501#discussioncomment-19401) you can create one yourself by fetching the status. Or also [this comment from another discussion](https://github.com/vercel/vercel/discussions/4824#discussioncomment-36061) has some additional ideas.

**Comment by whatjackhasmade** [comment]
I built my own mini-express server which you can clone and host on somewhere like Heroku for free.

Repository (with readme) - https://github.com/whatjackhasmade/vercel-status-express

e.g. My server - https://vercel-status.herokuapp.com/whatjackhasmade/wjhm-nextjs

Example badge - 

<img width="543" alt="Screenshot 2021-01-09 at 15 14 02" src="https://user-images.githubusercontent.com/11874178/104095227-a0687a00-528d-11eb-8b3d-1725c50245e5.png">

**Comment by ezralazuardy** [comment]
someone already made this happen. https://github.com/vercel/vercel/discussions/4824#discussioncomment-412794

**Reply by mdestafadilah** [reply]
not working.

**Comment by CodeLikeAGirl29** [comment]
I found this to be super helpful!

https://github.com/therealsujitk/vercel-badge

**Reply by sebastianoscarlopez** [reply]
Thanks 🥇
